issue_summarization_github_issues/setup_a_kubeflow_cluster.md

```diff
@@ -0,0 +1,174 @@
+# Setup Kubeflow
+
+In this part, you will setup kubeflow on an existing kubernetes cluster.
+
+## Requirements
```

Comment:
Can we just link to the Kubeflow user guide as opposed to duplicating setup instructions here?